src/components/Overlay.js

```diff
@@ -18,20 +18,23 @@ const Overlay = ({ children, onCloseOverlay, isOpen = false, className }) => {
 
   const ref = useRef(null);
 
-  const clickListener = useCallback((e) => {
-    if (ref.current.contains(e.target)) {
-      onCloseOverlay();
-    }
-  }, [ref.current])
+  const clickListener = useCallback(
+    (e) => {
+      if (ref.current.contains(e.target)) {
+        onCloseOverlay();
+      }
+    },
+    [ref.current]
```

Comment:
Looks like the linter is asking us to add one extra dependency for `useCallback`:

```suggestion
    [ref.current, onCloseOverlay]
```